app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -393,6 +401,80 @@ class HomeActivityTest {
     }
   }
 
+  // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
+  @RunOn(TestPlatform.ESPRESSO) // Incorrectly passes on Robolectric and shouldn't be re-enabled
+  @Config(qualifiers = "port-mdpi")
+  @Test
+  fun testHomeActivity_phonePort_longProfileName_welcomeMessageIsDisplayed() {
+    launch<HomeActivity>(createHomeActivityIntent(longNameInternalProfileId)).use {
```

Comment:
are we keeping `longNameInternalProfileId` for fixing as per the earlier comments.

Reply:
Rather than hardcoding the profileId, I created a variable.
Should I replace it with the hardcoded id?

Reply:
What I meant here is we already have 

```kotlin
private val internalProfileId: Int = 0
private val internalProfileId1: Int = 1
```

then why we need one more variable?

Reply:
We are using this in the [new test cases](https://github.com/oppia/oppia-android/pull/2396/files#r561593601)